src/mono/wasm/debugger/BrowserDebugProxy/DebugStore.cs

```diff
@@ -541,10 +541,19 @@ public unsafe AssemblyInfo(string url, byte[] assembly, byte[] pdb)
             asmStream = new MemoryStream(assembly);
             peReader = new PEReader(asmStream);
             asmMetadataReader = PEReaderExtensions.GetMetadataReader(peReader);
+            Name = asmMetadataReader.GetAssemblyDefinition().GetAssemblyName().Name + ".dll";
+            AssemblyNameUnqualified = asmMetadataReader.GetAssemblyDefinition().GetAssemblyName().Name + ".dll";
             if (pdb != null)
             {
                 pdbStream = new MemoryStream(pdb);
-                pdbMetadataReader = MetadataReaderProvider.FromPortablePdbStream(pdbStream).GetMetadataReader();
+                try
+                {
+                    pdbMetadataReader = MetadataReaderProvider.FromPortablePdbStream(pdbStream).GetMetadataReader();
+                }
+                catch (BadImageFormatException)
+                {
+                    Console.WriteLine($"Warning: Unable to read debug information of: {Name} (use DebugType=Portable/Embedded)");
```

Comment:
you could log this using https://chromedevtools.github.io/devtools-protocol/tot/Log/ have it surface to the frontend

Reply:
You emit https://chromedevtools.github.io/devtools-protocol/tot/Log/#event-entryAdded from the proxy?

Reply:
As far as I understood this is not supported by JSDebugger so I implemented using `Runtime.consoleAPICalled`.